examples/cameoparison-svelte-actors/src/app/machine.js

```diff
@@ -0,0 +1,38 @@
+import { actions, assign, createMachine, spawn } from 'xstate';
+const { stop } = actions;
+
+import { welcomeMachine } from '../welcome/machine.js';
+import { gameMachine } from '../game/machine.js';
+
+export const machine = createMachine({
+  id: 'appMachine',
+  context: {
+    welcomeActor: undefined,
+    gameActor: undefined
+  },
+
+  entry: assign({
+    welcomeActor: (context, event) => spawn(welcomeMachine, 'welcomeActor')
+  }),
+
+  initial: 'welcome',
+  states: {
+    welcome: {
+      on: {
+        PLAY: {
+          target: 'game',
+          actions: assign({
+            gameActor: (context, event) =>
+              spawn(gameMachine(event.data), 'gameActor')
+          })
+        }
+      }
+    },
+    game: {
+      on: {
+        GREET: 'welcome'
+      },
+      exit: stop('gameActor')
```

Comment:
while it might not matter here - maybe it would be a good idea to also remove this from the context?

Reply:
Absolutely good advice: calling `stop` and cleaning the `context` should go hand in hand.

Reply:
I thought I resolved this piece, but there are some warnings in the console now:
```
Warning: No implementation found for action type 'xstate.assign'
```
It was reported here: https://github.com/statelyai/xstate/issues/1792

The warning appears a couple of places, for example after playing the game it lands on the Feedback page, and when clicking on the back button the warning is logged.

I tried adding the `preserveActionOrder: true` thinking that the fixed order will help, but it did not, so I removed it. I am not sure if the `preserveActionOrder` is needed here or not.

Reply:
Found two different cases when the warning appeared for me:
- modelling: I defined `exit` action when the machine was not leaving that state ever -> fixed here: https://github.com/statelyai/xstate/pull/2539/commits/5409f9b0a6010e0b175eb4ca2c2b1451baa44cdd
- Vite HMR: when making some change in the source and the running app was reloaded